rts/lra/lra-client/src/main/java/io/narayana/lra/client/LRAInfoImpl.java

```diff
@@ -95,6 +86,24 @@ public boolean isTopLevel() {
         return this.isTopLevel;
     }
 
+
+    public long getStartTime() {
```

Comment:
I think that the time methods that have been added should be in some kind of a stats API rather than core LRA info. Not all implementations may want to / be able to report these accurately so perhaps move to an extended optional interface

Reply:
Issue was raised over here: https://issues.jboss.org/browse/JBTM-3042